### PR TITLE
TTree Cluster Enhancement v6-08

### PR DESCRIPTION
**Pull request by pcanal**

### CODE REVIEW

File: tree/tree/src/TBranch.cxx [hunk 1]
```diff
@@ -1807,20 +1810,30 @@ void TBranch::Print(Option_t*) const
...
    }
    Printf("*Baskets :%9d : Basket Size=%11d bytes  Compression= %6.2f     *",fWriteBasket,fBasketSize,cx);
+
+   if (strncmp(option,"basketsInfo",strlen("basketsInfo"))==0) {
+      Int_t nbaskets = fWriteBasket;
+      for (Int_t i=0;i<nbaskets;i++) {
+         Printf("*Basket #%4d  entry=%6lld  pos=%6lld  size=%5d",
+                i, fBasketEntry[i], fBasketSeek[i], fBasketBytes[i]);
+      }
+   }
```
Comment (amadio) [inline tree/tree/src/TBranch.cxx]: Please apply proper formatting according to coding conventions. Thanks.

**Comment by lmoneta** [comment]
@pcanal Is this PR still relevant ?

**Comment by pcanal** [comment]
@phsft-bot build

**Comment by pcanal** [comment]
Failure are unrelated.  Merging.

**Pull request merged**